client/cli/src/lib.rs

```diff
@@ -953,6 +1025,37 @@ impl FedimintCli {
                     .await?;
                 Ok(CliOutput::EpochCount { count })
             }
+            Command::Module { id, arg } => {
+                let cfg = cli.load_config_or_terminate();
+                let decoders = cli.load_decoders(&cfg, &self.module_gens);
+                let db = cli.load_db_or_terminate(&decoders);
+                let (_id, module_cfg) = match id {
+                    ModuleSelector::Id(id) => (
+                        id,
+                        cfg.as_ref()
+                            .get_module_cfg(id)
+                            .or_terminate_msg(CliErrorKind::IOError, "Can't load module"),
+                    ),
+                    ModuleSelector::Kind(kind) => cfg
+                        .as_ref()
+                        .get_first_module_by_kind_cfg(kind)
```

Comment:
So is getting by kind just to make the command more user-friendly?

Reply:
Yeap. I suspect in majority of cases people will have one module instance of a given kind, so referring to it by kind does seem to make things smoother ... .

We could possibly change it to be "by kind, only if it is not ambiguous" or something.